feature/posts/src/commonMain/kotlin/com/tunjid/heron/posts/State.kt

```diff
@@ -16,13 +16,16 @@
 
 package com.tunjid.heron.posts
 
+import com.tunjid.heron.data.core.models.Post
 import com.tunjid.heron.scaffold.navigation.NavigationAction
 import com.tunjid.heron.ui.text.Memo
 import kotlinx.serialization.Serializable
 import kotlinx.serialization.Transient
 
 @Serializable
 data class State(
+    @Transient
+    val posts: List<Post> = emptyList(),
```

Comment:
We will most likely be fetching a `TiledList<PostDataQuery, TimelineItem>` here so we can reuse the UI for those.